src/oneD/Sim1D.cpp

```diff
@@ -448,34 +448,34 @@ void Sim1D::solve(int loglevel, bool refine_grid)
 
                 if (loglevel > 6) {
                     save("debug_sim1d.yaml", "debug",
-                         "After successful Newton solve");
+                         "After successful Newton solve", true);
                 }
                 if (loglevel > 7) {
                     saveResidual("debug_sim1d.yaml", "residual",
-                                 "After successful Newton solve");
+                                 "After successful Newton solve", true);
                 }
                 ok = true;
             } else {
                 debuglog("    failure. \n", loglevel);
                 if (loglevel > 6) {
                     save("debug_sim1d.yaml", "debug",
-                         "After unsuccessful Newton solve");
+                         "After unsuccessful Newton solve", true);
                 }
                 if (loglevel > 7) {
                     saveResidual("debug_sim1d.yaml", "residual",
-                                 "After unsuccessful Newton solve");
+                                 "After unsuccessful Newton solve", true);
                 }
                 if (loglevel > 0) {
                     writelog("Take {} timesteps   ", nsteps);
                 }
                 dt = timeStep(nsteps, dt, m_state->data(), m_xnew.data(), loglevel-1);
                 m_xlast_ts = *m_state;
                 if (loglevel > 6) {
-                    save("debug_sim1d.yaml", "debug", "After timestepping");
+                    save("debug_sim1d.yaml", "debug", "After timestepping", true);
```

Comment:
A minor formatting nitpick: please wrap this line to keep the maximum line length no more than 88 characters.

Reply:
That line is 82 characters, is that just too close to the 88 limit?

Reply:
Sorry, my mistake. This is fine. I think I had something set incorrectly in my editor while looking at this.